lib/Elastica/Client.php

```diff
@@ -679,6 +680,16 @@ public function request($path, $method = Request::GET, $data = [], array $query
         return $response;
     }
 
+    public function requestEndpoint(AbstractEndpoint $endpoint)
```

Comment:
Could you add a comment to this method that it takes each endpoint from `elasticsearch-php`?